README.md

```diff
@@ -10,35 +10,25 @@ If you are looking for the loader which uses [EJS templating engine](https://git
 
 ## Usage
 
-[Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
+[Documentation: Using loaders](https://webpack.js.org/concepts/loaders/)
 
 ``` javascript
-var template = require("ejs!./file.ejs");
+var template = require("./file.ejs");
```

Comment:
Why did you remove the loader part? This example won’t work out of the box.

Reply:
Yes, it's not required if there is a config but for the sake of Readme I don't want to complicate things. Please revert.

Reply:
Please leave this part as it was